.gitignore

```diff
@@ -1,2 +1,3 @@
 config.cfg
-.idea/*
\ No newline at end of file
+.idea/*
+.venv/*
```

Comment:
Looks good but could use a newline at the end.

Reply:
Sure thing, there wasn't one originally which was why I didn't pay much attention to there not being one on my commit either.